### PR TITLE
Modified React and Docker descriptions.

### LINKED ISSUE #3: React and Docker descriptions could be less abrasive.
It's nice not to belittle people for not knowing things. Stuff like this makes this industry seem less welcoming to newbies, especially women and other unrepresented groups.

### PR DESCRIPTION
**Pull request by irikuo**
Fixes #3 :)

### CODE REVIEW

**Comment by reevoo-samuel** [comment]
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **designed for change**, **easy to understand**
and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [ ] Description explains what you have done and why.
- [ ] Tests have been written (or you have explained why you don't need
  tests).
- [ ] Documentation/comments have been added/updated.

**For the Reviewer**
- [ ] The code is well-designed and able to cope with change.
- [ ] The code is easy to understand.
- [ ] Tests cover happy and unhappy paths in the code.
- [ ] Tests are green on CI (or pulled/tested on your machine).

**Comment by reevoo-samuel** [comment]
I don't see any comments on your Pull Request.

Are you too good for code reviews now?
No. No you aren't.

Maybe you forgot to say who reviewed it. That's fine, but let me know.

File: README.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ A collection of the awesome libraries, frameworks, tools, resources and other be
 ## Infrastructure
 
 - Containers
-  - [Docker](https://www.docker.com/) - Don't you know what Docker is? Really?
+  - [Docker](https://www.docker.com/) - With Docker you can create a container with all the configuration and tools that you need and deploy your app to your staging or production environments. You don't need to worry never again about inconsistent versions in your different environments. Isn't that cool?
   - [Kubernetes](https://kubernetes.io/) - Kubernetes is an open-source system for automating deployment, operations, and scaling of containerized applications, oh and it's awesome.
```
Comment (jelamo) [inline README.md:30]: You don't need to worry ever again

**Comment by jelamo** [comment]
LGTM

**Comment by lpil** [comment]
:+1:

**Pull request merged**